src/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess/Activities/CalculateDaysinLearning.cs

```diff
@@ -21,9 +21,9 @@ public CalculateDaysInLearning(ICommandDispatcher commandDispatcher, ILogger<Cal
         [FunctionName(nameof(CalculateDaysInLearning))]
         public async Task Create([ActivityTrigger] CalculateDaysInLearningInput input)
         {
-            _logger.LogInformation("Calculating DaysinLearning for apprenticeship incentive id {apprenticeshipIncentiveId}, active period {activePeriod}", input.ApprenticeshipIncentiveId, input.ActivePeriod);
+            _logger.LogDebug("Calculating DaysinLearning for apprenticeship incentive id {apprenticeshipIncentiveId}, active period {activePeriod}", input.ApprenticeshipIncentiveId, input.ActivePeriod);
```

Comment:
Shouldn't the log be in a decorator of the  logger be a decorator of the _commandDispatcher?